source/server/api/v1/_tests/appRoutes.test.js

```diff
@@ -11,10 +13,10 @@ describe( 'app routes', () => {
         .get( '/' )
         .then( response => {
           const responseHTML = response.text
-          const expectedHTML = '<!DOCTYPE html>\n<html>\n  <head>\n    <link rel="stylesheet" type="text/css" href="main.css" />\n  </head>\n  <body>\n    <div id="anchor"></div>\n    <script src="index.js"></script>\n  </body>\n</html>\n'
+          const expectedHTML = fs.readFileSync(path.join(__dirname, '/../../../../../public/index.html'), 'utf8')
```

Comment:
I'd rather the path be stored in a variable so this line looks sexier, also pregnant parens plz:
```js
const pathToHTML = path.join ( __dirname, '/../../../../../public/index.html' )
const expectedHTML = fs.readFileSync( pathToHTML, 'utf8' )
```